cql3/expr/restrictions.cc

```diff
@@ -7,9 +7,13 @@
  */
 
 #include "cql3/statements/request_validations.hh"
+#include "exceptions/exceptions.hh"
+#include "schema/schema.hh"
```

Comment:
Are those includes needed?
Both `exceptions` and `schema` were already used in the code previously.

Reply:
they are not needed. but they are added for the reason of: it would be better if we do not rely on the headers to include the declarations on behalf of us. for instance, considering following scenario, the author of a header `foo.hh`, which defines `foo`, and this header file is quite popular in the project.  he/she wants to clean it up, by removing some unused headers from it. but the compiler fails to compile the tree after this change. because lots of source files actually rely on the "unused headers", so he/she just gives up. despite that this would otherwise be a legitimate change. and probably would help to improve the compiling speed. so, i think it would be better to include the used headers explicitly, if the header is not supposed to include the referenced declarations / definition.

Reply:
Makes sense, let's keep them then.